ts/components/session/settings/SessionSettings.tsx

```diff
@@ -350,6 +352,23 @@ class SettingsViewInner extends React.Component<SettingsViewProps, State> {
           okTheme: SessionButtonColor.Danger,
         },
       },
+      {
+        id: 'audio-message-autoplay-setting',
+        title: window.i18n('audioMessageAutoplayTitle'),
+        description: window.i18n('audioMessageAutoplayDescription'),
+        hidden: false,
+        type: SessionSettingType.Toggle,
+        category: SessionSettingCategory.Appearance,
+        setFn: () => {
+          window.inboxStore?.dispatch(toggleAudioAutoplay());
+        },
+        content: {
+          defaultValue: window.inboxStore?.getState().userConfig.audioAutoplay,
```

Comment:
this should be a reducer. Can you move the SessionSettings to hooks or it's too much work?